client/src/components/ModalDialog/ModalDialog.module.css

```
@@ -0,0 +1,44 @@
+.modalWrapper.displayed {
+  display: block;
+}
+
+.modalWrapper.hidden {
+  display: none;
+}
+
+.scrim {
+  position: fixed;
+  z-index: 9999;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: all;
+  background: rgba(0, 0, 0, 0.25);
+}
+
+.modal {
+  background: white;
+  position: absolute;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 40px 80px;
+  border-radius: 10px;
```

Should be 20px per #74.
```suggestion
  border-radius: 20px;
```